app/actions/home.js

```diff
@@ -128,6 +128,26 @@ export function getSpotifySearch(searchKey){
 	}
 }
 
+export function searchArtist(artist){
+	return(dispatch,getState)=>{
+		let state = getState();
+		axios({
+		  method:'get',
+		  url:'/api/v1/spotify?artist='+artist
+		}).then(res=>{
+			dispatch(batchActions([
+				RoutingActions.locationChange(`/search/${artist}/${state.player.id}`),
```

Comment:
if we reload the page, how do we know that it was an Artist Search? We can append to search something like a-{value} for artist, s-{value} for song and p-{value} for playlist ?